less/_popups.less

```diff
@@ -1,25 +1,19 @@
 .popover, .tooltip-inner {
-    background: #323232;
-    color: #FFF;
-    border-radius: 2px;
-
+  color: #ececec;
```

Comment:
may you use color variables instead? Feel free to add them to _variables.less, just be sure to give them a meaningful name.